shopify_app.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('minitest')
   s.add_development_dependency('mocha')
 
-  s.files         = `git ls-files`.split("\n")
+  s.files         = `git ls-files`.split("\n").reject { |f| f.match(%r{^(test|example)/}) }
```

Comment:
@celsodantas made a good point that adding this would increase the gem size. I added a reject block here to keep some files out of the gem when it is built